man/confchpl.rst

```diff
@@ -1,5 +1,5 @@
 
-:Version: 2.1
+:Version: 2.2.0 pre-release
```

Comment:
Should this still have just major and minor version numbers like `2.1` did?

Reply:
Yes, will fix...